test/cpp/qps/driver.cc

```diff
@@ -315,11 +317,18 @@ std::unique_ptr<ScenarioResult> RunScenario(
     if (!servers[i].stream->Read(&init_status)) {
       gpr_log(GPR_ERROR, "Server %zu did not yield initial status", i);
     }
-    gpr_join_host_port(&cli_target, host, init_status.port());
-    client_config.add_server_targets(cli_target);
-    gpr_free(host);
-    gpr_free(driver_port);
-    gpr_free(cli_target);
+    if (qps_server_target_override != NULL && strlen(qps_server_target_override) > 0) {
```

Comment:
Partly I was trying to avoid host-port parsing, but mostly the benefit was to allow configuring of the qps server target when invoking the driver, rather than from info provided by the worker. I could change the parameter to a list?